client/test/characterData.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { allCharactersList } from "@hidden-identity/shared";
+import { describe, expect, test } from "vitest";
+
+describe("characterData", () => {
```

Comment:
Went through and made all reminders have unique names, that way we can just look them up on a role easily.  Wrote some tests to ensure the data is valid